src/core/execute.h

```diff
@@ -91,6 +91,7 @@ typedef enum ExecCommandFlags {
         EXEC_COMMAND_FULLY_PRIVILEGED = 1 << 1,
         EXEC_COMMAND_NO_SETUID        = 1 << 2,
         EXEC_COMMAND_AMBIENT_MAGIC    = 1 << 3,
+        EXEC_COMMAND_NO_ENV           = 1 << 4,
```

Comment:
hmm, i think this should probably be named "DONT_ENV_EXPAND" or so? `NO_ENV` is a bit misleading I think. People might misunderstand that setting this flag means we'd no pass any env vars to the executed process, but this is not what this is about here...

Reply:
Yeah I originally had this as NO_ENV_EXPAND but changed it to look more like the string on the dbus API. I guess there's no harm in making them both have the word expand or being a little mismatched.